bin/pulsar

```diff
@@ -70,7 +71,8 @@ where command is one of:
     broker              Run a broker server
     bookie              Run a bookie server
     zookeeper           Run a zookeeper server
-    global-zookeeper    Run a global-zookeeper server
+    global-zookeeper    Deprecated. Run a configuration-store server
```

Comment:
probably better to leave original text and append "deprecated. use 'configuration-store' instead".

`Run a global zookeeper server (Deprecated. Use "bin/pulsar configuration-store" instead)`